packages/nvidia_nat_test/src/nat/test/tool_test_runner.py

```
@@ -137,6 +144,22 @@ def get_function_config(self, name: str) -> FunctionBaseConfig:
 
+    async def add_function_group(self, name: str, config: FunctionGroupBaseConfig) -> FunctionGroup:
+        """Mock implementation - not used in tool testing."""
+        raise NotImplementedError("Mock implementation does not support add_function_group")
+
+    def get_function_group(self, name: str) -> FunctionGroup:
+        """Return a mock function group if one is configured."""
+        if name in self._mocks:
+            mock_fn_group = MagicMock(spec=FunctionGroup)
+            mock_fn_group.ainvoke = AsyncMock(return_value=self._mocks[name])
+            return mock_fn_group
+        raise ValueError(f"Function group '{name}' not mocked. Use mock_function_group() to add it.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)